docs/platform/SEPES component overview.md

```diff
@@ -32,11 +35,13 @@ Protocol: https
 
 Coordinate the different services, and make actions on the behalf of the user.
 
-Tech Stack: dot.net, asp.net core, C#
+Tech Stack: .net, asp.net core, C#
 
 Maturity:
 
-Protocol: Uses https rest protocol and can be configured to use http. http should only be used for cases that requires it.
+Protocol: Uses https rest protocol and can be configured to use http. http should only be used for cases that requires it. 
+
+Its only safe its behind a proxy that handle HTTPS terminations. Or in local development setting. Docker compose have nothing to do with protocol on it own. Proxy is the important keyword.
```

Comment:
Its only safe its behind a proxy that handle HTTPS terminations. Or in local development setting. ~~Docker compose have nothing to do with protocol on it own. Proxy is the important keyword.~~